refresh/fromswagger/swaggerize.js

```diff
@@ -196,8 +185,11 @@ exports.parse = function (memfs, swaggerPath) {
   debug('in parse')
   return loadAsync(memfs, swaggerPath)
     .then(loaded => {
-      ensureValid(loaded, swaggerPath)
-      debug('successfully validated against schema')
-      return { loaded: loaded, parsed: parseSwagger(loaded) }
+      var loadedAsJSONString = JSON.stringify(loaded)
```

Comment:
A comment would be good for this line (or the parse below) about the need to clone the Swagger object because of the destructive nature of the validation (assuming that is what is going on here).